app/README.md

```diff
@@ -49,7 +49,9 @@ From the `app/` directory:
 
 Alternatively, you can run the application in a Docker container:
 
-1. From the root directory run `docker-compose up -d --build`
+1. From the root directory run `docker compose up -d --build`
+
+***NB.*** This template uses [Docker Compose V2](https://www.docker.com/blog/announcing-compose-v2-general-availability/) (viz. `docker compose` instead of `docker-compose`).
```

Comment:
nit: What's `NB`? I dunno if this line is necessary if it's likely that folks will have the necessary Docker version installed already.

Reply:
Yeah I left a similar comment on the flask template PR

Reply:
As an aside, this file should not have any mention of the word "template" since this file should make sense in the context of the project.